packages/nodejs/scripts/extension/extension.js

```diff
@@ -6,13 +6,13 @@ const fs = require("fs")
 const crypto = require("crypto")
 const childProcess = require("child_process")
 
-const { AGENT_VERSION, MIRRORS, TRIPLES } = require("./extension/constants")
+const { AGENT_VERSION, MIRRORS, TRIPLES } = require("./support/constants")
```

Comment:
Can you also send in a PR for the agent repo to update this path?

Reply:
PR: https://github.com/appsignal/appsignal-agent/pull/738